internal/provider/resource_aci_annotation.go

```diff
@@ -81,6 +81,7 @@ func (r *TagAnnotationResource) Schema(ctx context.Context, req resource.SchemaR
 				Required: true,
 				PlanModifiers: []planmodifier.String{
 					stringplanmodifier.UseStateForUnknown(),
+					SetToStringNullWhenStateIsNullPlanIsUnknownDuringUpdate(),
```

Comment:
do we need this for required attributes?

Reply:
I added this here because of stringplanmodifier.UseStateForUnknown() exists. I am unsure how new and required attributes not available on older versions of APIC will behave when running tests on those older versions.